identify/identify.py

```diff
@@ -159,7 +159,9 @@ def parse_shebang(bytesio):
     cmd = tuple(_shebang_split(first_line.strip()))
     if cmd and cmd[0] == '/usr/bin/env':
         cmd = cmd[1:]
-        if cmd and cmd == 'nix-shell':
+        if cmd == ('nix-shell',):
+            if cmd[1:]:
+                return ()
```

Comment:
This is bullshit...

Reply:
solved in https://github.com/chriskuehl/identify/pull/130/commits/be23107269babb2a604437fca7f9a80ba8094411 &mdash; please validate and resolve.